test-data/callx.yaml

```diff
@@ -140,7 +140,7 @@ post:
   - r8.uvalue=2
 
 messages:
-  - "0: Invalid type (r3.type in {stack, packet})"
+  - "0: Invalid type (r3.type in {stack, packet, shared})"
```

Comment:
also copy the new test cases into this file, e.g., "bpf_map_update_elem with numeric stack value"

Reply:
Done.